app/models/service_template.rb

```diff
@@ -100,7 +100,7 @@ def self.with_tenant(tenant_id)
   end
 
   def self.with_additional_tenants
-    includes(:service_template_tenants => :tenant)
+    references(table_name.to_sym, :tenants).includes(:service_template_tenants => :tenant)
```

Comment:
Isn't this `to_sym` pointless since this just has `.to_s` called on it as you mentioned in the PR description anyway?

[[ref](https://github.com/rails/rails/commit/a84a20e1cd2133932dd1bd33f862aa2605cc821e)]